ci/npm-publish.yml

```diff
@@ -47,4 +47,4 @@ jobs:
       - run: npm ci
       - run: npm publish
         env:
-          NODE_AUTH_TOKEN: ${{secrets.GITHUB_TOKEN}}
+          NODE_AUTH_TOKEN: ${{secrets.GITHUB_TOKEN}}
```

Comment:
if there's no difference between the above lines, you can skip this file from the PR.